src/RpcServer/RpcServer.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <PackageId>Neo.Plugins.RpcServer</PackageId>
     <RootNamespace>Neo.Plugins</RootNamespace>
+    <TargetFramework>net6.0</TargetFramework>
+    <LangVersion>10</LangVersion>
```

Comment:
Why downgrade to .net6.0?

Reply:
These changes should not have been included. I've reverted them.